addons/website_profile/static/src/scss/website_profile.scss

```
@@ -125,6 +125,18 @@ $owprofile-color-bg: mix($body-bg, #efeff4);
 
     .o_pc_overlay {
         @include o-position-absolute(0,0,0,0);
+        // Allow hover effect on svg paths in order to display bootstrap tooltip,
+        // by pushing the absolute div behind it
+        z-index: -1;
+
+        // Only display single line for the next rank, and display ellipsis if it can not fit in the card
+        .o_pc_text_overflow {
+            @include o-text-overflow(-webkit-box);
+            width: 60%;
+            white-space: normal;
+            -webkit-box-orient: vertical;
+            -webkit-line-clamp: 1;
+        }
```

Last small comment and we are good to good after this.
Is there a reason why you use `-webkit` here (is there a specific case you need this)? As I see, I don't think that's mandatory and once set the display will not be the same between chrome and firefox for example.
Does it not work with just this ?
```scss
.o_pc_text_overflow {
    @include o-text-overflow;
    width: 60%;
}
```

---

Hum, then I think that we could do this. What do you think ?
```suggestion
        .o_pc_text_overflow {
            &:not(.o_editable) {
                @include o-text-overflow;
            }
            width: 60%;
        }
```

---

Changes are done but i have one question,
If we write a long sentence, it looks like 
https://www.awesomescreenshot.com/image/31638037?key=d636192a370d53204e58c0611fe31868

Will it be okay? what do you think?  

cc: @LNAodoo